### ISSUE TITLE
Next + Create React App

### ISSUE THREAD

**Issue by lhguerra** [lksilva/playing-with-nextjs#10]
A gente reparou que você fez o Next rodar em cima da estrutura do Create React App. Você teve alguma razão especial para isso?

**Comment by lksilva** [comment]
Sim! O create-react-app já cria um projeto React completo e todo configurado, o que facilita na hora de iniciar um projeto do zero.

**Comment by lhguerra** [comment]
Então, é que só com o Next também já vem bastante coisa pronta, é só criar uma page e sair codando (: 

Esse mini-curso explica isso https://learnnextjs.com/basics
